docs/tasks/federation/set-up-placement-policies-federation.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Set up placement policies in Federation
+redirect_from:
+- "/docs/tutorials/federation/set-up-placement-policies-federation/"
+- "/docs/tutorials/federation/set-up-placement-policies-federation.html"
+---
+
+{% capture overview %}
+
+This page shows you can enforce policy-based placement decisions over Federated
```

Comment:
suggested edit: "This page demonstrates how to enforce policy-based placement decisions..."